.circleci/config.yml

```diff
@@ -1,10 +1,18 @@
-version: 2
-
-references:
-  container_config: &container_config
+version: 2.1
+executors:
+  infrastructure_container_unstable:
+    docker:
+      - image: aeternity/infrastructure:master # should be changed to v1.8.0 after releasing
```

Comment:
Not sure what this comment it for, we should keep it master here

Reply:
right wrong line ;]